webapp/sources/rudder/rudder-web/src/main/webapp/secure/configurationManager/techniqueEditor.html

```diff
@@ -65,7 +76,18 @@
   });
 
 
+  app.ports.pushUrl.subscribe(function(id) {
 
+    var url = "/rudder/secure/configurationManager/techniqueEditor"
```

Comment:
Shouldn't we use `contextPath + "/secure/configurationManager/techniqueEditor"` instead of "/rudder/secure..." ?

Reply:
YEs !